Assets/Engine/Inventory/UI/ContainerUi.cs

```diff
@@ -55,6 +56,25 @@ private void UpdateContainer(Container container)
             Vector3[] v = new Vector3[4];
             rect.GetLocalCorners(v); 
             containerName.transform.localPosition = v[1] + new Vector3(0.03f * width, -0.02f * height, 0);
+
+            if (attachedContainer.GetComponent<Item>() != null)
+            {
+                Item item = attachedContainer.GetComponent<Item>();
+                item.ItemContainerChanged += OnItemContainerChanged;
```

Comment:
Make sure to unsubscribe from this event in `Destroy`, otherwise this object will keep existing, even when the UI is closed.